model.py

```diff
@@ -2965,8 +2965,8 @@ class Edition(Base):
     FULFILLABLE_MEDIA = [BOOK_MEDIUM]
 
     medium_to_additional_type = {
-        BOOK_MEDIUM : u"http://schema.org/Book",
-        AUDIO_MEDIUM : u"http://schema.org/AudioObject",
+        BOOK_MEDIUM : u"http://schema.org/EBook",
+        AUDIO_MEDIUM : u"http://bib.schema.org/AudioBook",
```

Comment:
This should be http://bib.schema.org/Audiobook - the 'B' isn't capitalized.

Reply:
Thanks, I'm always confused that EBook has intercaps and Audiobook doesn't.